### DISCUSSION TITLE
set PIPELINES_TELEMETRY_OPT_OUT in Gruntwork Pipelines?

### DISCUSSION THREAD

**Discussion by jeffreymlewis** [gruntwork-io/knowledge-base#788] [How do I] [answered]
I'm implementing Gruntwork Pipelines and would like to pass `PIPELINES_TELEMETRY_OPT_OUT=true` to the `pipelines` binary, to opt out of sharing "telemetry" with Gruntwork. There doesn't seem to be a way to do with the current GH Actions in the [pipelines-orchestrate](https://github.com/gruntwork-io/pipelines-orchestrate) repo. Is my only option forking this repo?

Also, can you tell me what "telemetry" the `pipelines` binary is sending to Gruntwork? I would check myself, but this binary seems to be closed source. ([link](https://github.com/gruntwork-io/pipelines-cli/issues/1))

---

<ins datetime="2023-12-04T21:06:30Z">
  <p><a href="https://support.gruntwork.io/hc/requests/110615">Tracked in ticket #110615</a></p>
</ins>

**Comment by oredavids** [comment, marked as answer]
@jeffreymlewis please see our [docs](https://docs.gruntwork.io/pipelines/data-collection/) on how to opt out and what data we collect. Also you are correct that the Pipelines CLI is a closed source Gruntwork product. We will be making that clear in that repository's documentation soon.
